build.sh

```diff
@@ -47,12 +43,31 @@ elif [ "$1" = "release" ]; then
   echo ""
   echo "Now sign ooniprobe_checksums.txt and upload it along with tarballs to GitHub"
 
+elif [ "$1" = "__docker" ]; then
+  set -x
+  shift
+  docker build -t oonibuild .
+  docker run -v `pwd`:/oonibuild                                               \
+             -w /oonibuild                                                     \
+             -t                                                                \
+             --cap-drop=all                                                    \
+             --user `id -u`:`id -g`                                            \
+             -e 'GOCACHE=/oonibuild/testdata/gotmp/cache'                      \
+             -e 'GOPATH=/oonibuild/testdata/gotmp/path'                        \
+             -e "TRAVIS_JOB_ID=$TRAVIS_JOB_ID"                                 \
+             -e "TRAVIS_PULL_REQUEST=$TRAVIS_PULL_REQUEST"                     \
+             oonibuild "$@"
```

Comment:
This is the new `__docker` sub command. Note that we pass along the environment variables that `goveralls` will use to submit the coverage information to Coveralls.